docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1214,6 +1222,21 @@ In other words the extension will use whatever cluster `kubectl` uses. The same
 
 At the moment no additional options are provided for further customization.
 
+=== Remote Debugging
+
+To remotely debug applications that are running on a cloud environment, we need to deploy the application as described in the previous section and additionally appending the property `quarkus.kubernetes.remote-debug.enabled=true`. This property will automatically configure the Java application to append the java agent configuration (for example: `-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005`) and also the service resource to listen the java agent port. 
+
+After your application has been deployed with the debug enabled, next you need to port-forward the service java agent port:app-name:
+
+[source,bash,subs=attributes+]
+----
+kubectl port-forward svc/<application name> 5005:5005
+----
+
+Using this command, you'll be forwaring all the TCP traffic from `localhost:5005` to the service name with port 5005 which is the one that the java agent uses by default for remote debugging. You can also configure another java agent port using the property `quarkus.kubernetes.remote-debug.address-port`.
+
+Finally, all you need to do is to configure your favorite IDE to attach the java agent process that is forwarded to `localhost:5005` and start debugging your application. For example, in IntelliJ IDEA, you can follow https://www.jetbrains.com/help/idea/tutorial-remote-debug.html:[this tutorial] to debug remote applications.
```

Comment:
`and start debugging your application` --> `... and start to debug your application.`

Reply:
changed